### PR TITLE
Replace .min.css with .css /1

### LINKED ISSUE #26180: remove all ".min" from layout files
See #26176

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #26180

### CODE REVIEW

File: aspnetcore/blazor/components/prerendering-and-integration.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ Update the imported layout file (`_Layout.cshtml`) to include the **`Client`** p
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 -   <title>@ViewData["Title"] - {APP NAME}</title>
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/components/prerendering-and-integration.md:213]: Diff markup doesen't look so great here
```diff
<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-   <title>@ViewData["Title"] - {APP NAME}</title>
+   <title>@ViewData["Title"] - BlazorHosted</title>
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="~/css/site.css" />
+   <link href="css/app.css" rel="stylesheet" />
-   <link rel="stylesheet" href="~/RPDonor.styles.css" asp-append-version="true" />
+   <link rel="stylesheet" href="BlazorHosted.Client.styles.css" asp-append-version="true" />
+   <component type="typeof(HeadOutlet)" render-mode="WebAssemblyPrerendered" />
</head>
```

**Comment by guardrex** [comment]
None of the Blazor articles/files/samples/examples are broken AFAIK. Let's not change the Blazor-related markup because the display of code is usually exactly what the dev will see from a project template when they open a file that they're being instructed to change.

**Comment by guardrex** [comment]
I checked on the .NET 6 bits for Blazor, and the minified assets are the ones that are used. Yes, please remove the Blazor updates from this PR. This is going to break both Blazor WASM and Blazor Server apps.

**Comment by Rick-Anderson** [comment]
@guardrex does Blazor override .gitignore and check in the min files?
WHen I do a global search and replace in VS Code, how do I exclude Blazor files?

**Comment by guardrex** [comment]
Yes, that's right ... 

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/.gitignore

In VSC, use the **files to exclude** filter with a path to the blazor folder ... that should factor out all of the Blazor stuff.

WRT debugging, there is no debugging of non-min files with Blazor, per se, because the size of such files unminified is a killer on top of all of the assets that must be downloaded to the client. The project templates don't have unminified assets. For example for WASM ...

https://github.com/dotnet/aspnetcore/tree/main/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/bootstrap

**Pull request closed without merging**